frontend/src/components/Menu/DropdownMenu.stories.tsx

```diff
@@ -0,0 +1,171 @@
+import {
+  Box,
+  SimpleGrid,
+  Table,
+  Tbody,
+  Td,
+  Text,
+  Th,
+  Thead,
+  Tr,
+} from '@chakra-ui/react'
+import { Meta, Story } from '@storybook/react'
+
+import { ThemeButtonVariant } from '~theme/components/Button'
+
+import { DropdownMenu } from './DropdownMenu'
+
+export default {
+  title: 'Components/DropdownMenu',
+  component: DropdownMenu,
+} as Meta
+
+type DropdownMenuTemplateProps = {
+  isFullWidth?: boolean
+  isOpen?: boolean
+  variant: ThemeButtonVariant
+  children: string
+}
+
+type DropdownMenuGroupTemplateProps = {
+  variant: ThemeButtonVariant
+}
+
+const { DropdownMenuButton, DropdownMenuItem, DropdownMenuList } = DropdownMenu
```

Comment:
nit: we should not need to destructure from the main component, should just use `DropdownMenu.Button` to instantiate the component directly, e.g.

```tsx
<DropdownMenu>
  <DropdownMenu.Button />
  <DropdownMenu.List>
    <DropdownMenu.Item/>
    <DropdownMenu.Item/>
  </DropdownMenu.List>  
</DropdownMenu>

Reply:
thanks! edited